### ISSUE TITLE
Recursive function for batch tool

### ISSUE THREAD

**Issue by Encryption767** [kmkolasinski/AwesomeBump#129]
I wasn't able to find any option to recursively convert textures from a directory with the batch tool. 

If I have missed this option then where can I find it and if not then will it be added in the future?

**Comment by kmkolasinski** [comment]
Yes this feature works for flat folder structure. If you have multiple folders inside it's not possible for now. Don't think if anyone is going to implement this in future. The simplest solution is to copy all the images to one folder and run batch tool.
